### ISSUE TITLE
Failed to connect ThorLabs

### ISSUE THREAD

**Issue by waswis** [dorssel/usbipd-win#943]
Hi, 

I tried to attach ThorLabs PM100USB from windows to WSL2 but seems like I'm going to a dead end. For some reason, I cannot see PM100USB connected to ubuntu using lsusb even in powershell it says that the device is attached to WSL. Can you help me with this?

Many thanks,
Wastu

**Comment by dorssel** [comment]
What is the output of (WSL-side) `lsusb` after attaching?

**Comment by waswis** [comment]
![image](https://github.com/dorssel/usbipd-win/assets/168958230/0fc480fe-bd46-492c-b0c6-b15544b6c5e4)

**Comment by waswis** [comment]
No, I'm using the same account for admin and local account (checked under control panel -> user account)

**Comment by dorssel** [comment]
> in powershell it says that the device is attached to WSL

Which command? What is the output? Which version of `usbipd-win` are you using?

**Comment by waswis** [comment]
Hi!

This is the version of the usbipd-win I am using: 4.2.0+54.Branch.master.Sha.5b3...

I want to attach the PM100USB to WSL (already shared using bind)
![image](https://github.com/dorssel/usbipd-win/assets/168958230/8bc13d18-8d81-4f70-89d2-c8b77a6ddc89)

After attaching using attach
![image](https://github.com/dorssel/usbipd-win/assets/168958230/6e9015dd-e88e-441a-872e-a77c0824adbd)

Checking on my WSL
![image](https://github.com/dorssel/usbipd-win/assets/168958230/cd08e1ec-a561-4e48-966d-d8aaba22c784)

**Comment by dorssel** [comment]
Looks like the device is detaching immediately again. Can you follow the troubleshooting at https://github.com/dorssel/usbipd-win/wiki/Troubleshooting?
